### DISCUSSION TITLE
how to reset or clear username/password on viseron web

### DISCUSSION THREAD

**Discussion by ceewanna** [roflcoopter/viseron#655] [General]
The first time I log on to localhost:XXXX for viseron I get asked to set username and password. If I get access to the page from other local nodes on the same network, it would simply ask for username and password.

Currently, I could not locate where I could make change for password. What if I happen to forget the username/password, how do I reset or clear it?

**Comment by roflcoopter** [comment]
At the moment the only way to do that is to delete the `/config/.viseron/auth` and `/config/.viseron/onboarding` files and then register a new user.

I will make this easier once I implement the ability to add multiple users!

**Comment by ceewanna** [comment]
ok. Thanks.
